src/applications/disability-benefits/686/tests/00.686-dependent-status.e2e.spec.js

```diff
@@ -11,4 +11,4 @@ module.exports = E2eHelpers.createE2eTest((client) => {
   client.end();
 });
 
-module.exports['@disabled'] = !manifest.production;
+module.exports['@disabled'] = manifest.e2eTestsDisabled;
```

Comment:
An unfortunate consequence of decoupling the content directory from the manifests is that manifest's `production` property is only useful to Webpack code. To make that more clear, I renamed it to `e2eTestsDisabled`.

I think this value could be derived by checking the `settings.js` and seeing that there are no pages registered to the manifest's entry name, but our e2e tests in general will need to be revisited, as they sort of sit outside of both app-code and content.